### ISSUE TITLE
Multiplicity calculation wrong in MillipedeLikelihood (Trac #793)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#787]
MillipedeLikelihood::GetMultiplicity() returns the total number of terms in the likelihood. This is wrong if more than 1 term involves a bin with zero entries. Since every way of grouping the bins with zero entries together results in the same likelihood, the multiplicity should be the number of nonzero bins + 1 if any bins are empty, and just the number of nonzero bins otherwise.

Migrated from https://code.icecube.wisc.edu/ticket/793
```json
{
    "status": "closed", 
    "changetime": "2015-02-12T10:48:02", 
    "description": "MillipedeLikelihood::GetMultiplicity() returns the total number of terms in the likelihood. This is wrong if more than 1 term involves a bin with zero entries. Since every way of grouping the bins with zero entries together results in the same likelihood, the multiplicity should be the number of nonzero bins + 1 if any bins are empty, and just the number of nonzero bins otherwise.", 
    "reporter": "jvansanten", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1423738082255104", 
    "component": "combo reconstruction", 
    "summary": "Multiplicity calculation wrong in MillipedeLikelihood", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2014-10-23T19:00:01", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T21:17:14: 
* nega changed owner from "" to "jvansanten"
* nega changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150212T10:48:02: 
* jvansanten commented:

> In changeset 128985:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="128985"
> Correct multiplicity calculation in MillipedeLikelihood to account for empty
> bins (fixes #787)
> ```


* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "assigned" to "closed"
